website_compassion/models/correspondence_template.py

```diff
@@ -15,10 +15,15 @@ class CorrespondenceTemplate(models.Model):
     website_published = fields.Boolean()
     template_image_url = fields.Char(compute="_compute_template_image_url")
 
-    def _compute_template_image_url(self):
+    def _compute_template_image_url(self, resize_factor=10):
         web_base_url = (self.env["ir.config_parameter"].sudo()
                         .get_param("web.external.url"))
+
+        def resize_dimension(template):
+            return "x".join(list(map(lambda dim: str(round(dim / resize_factor)),
+                                     template.get_template_size(1))))
```

Comment:
```suggestion
        def resize_dimension(template):
            dim = template.get_template_size(1/resize_factor)
            return f'{round(dim[0])}x{round(dim[1])}'
```

Reply:
Or maybe we can set resize_factor = 0.1 and put `template.get_template_size(resize_factor)`